### PR TITLE
new technote about magick and knitr

### PR DESCRIPTION
**Pull request by jeroen**
Short tech note to announce small update: [preview link](https://deploy-preview-121--ropensci.netlify.com/technotes/2017/11/07/magick-knitr/)

### CODE REVIEW

File: content/technotes/2017-11-07-magick-knitr.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    image_composite(fig, banana, offset = "+200+200")
+  }) %>% 
+  image_resize("50%") %>% 
+  image_animate()
+```
+
+![fig4](https://i.imgur.com/mi67gjt.gif)
+
+
+The magick vignette [source code](https://raw.githubusercontent.com/ropensci/magick/master/vignettes/intro.Rmd) is itself writte in Rmarkdown, so it's great example to see this in action. Try rendering it in RStudio to see how easy it is!
```
Comment (sckott) [inline content/technotes/2017-11-07-magick-knitr.md]: "written"

File: content/technotes/2017-11-07-magick-knitr.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+## Magick in Shiny Apps
+
+While we're at it, several people had asked how to use magick images in shiny apps. The easiest way is to write the image to a `tempfile()` within the `renderImage()` callback function For example the server part could look like this:
```
Comment (sckott) [inline content/technotes/2017-11-07-magick-knitr.md]: > ... callback function. For example ...

File: content/technotes/2017-11-07-magick-knitr.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+categories: technotes
+cover: "https://i.imgur.com/tTFk7ig.jpg"
+tags:
+- R
+- packages
+- magick
+- images
+---
+
+This week [magick](https://cran.r-project.org/web/packages/magick/vignettes/intro.html) 1.5 appeared on CRAN. The latest update adds support for using images in knitr documents and shiny apps. In this post we show how this nicely ties together a reproducible image workflow in R, from source image or plot directly into your report or application.
```
Comment (sckott) [inline content/technotes/2017-11-07-magick-knitr.md]: Did you want to link to vignette here? or did you mean to go to cran pkg landing page?

Comment (jeroen) [reply content/technotes/2017-11-07-magick-knitr.md]: Yes the vignette.

**Pull request merged**